### PR TITLE
create allergyintolerance-create.md

### PR DESCRIPTION
**Pull request by isaacvetter**

### CODE REVIEW

File: docs/hooks/allergyintolerance-create.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+## Context
+
+Field | Optionality | Prefetch Token | Type | Description
+----- | -------- | ---- | ---- | ----
+`userId` | REQUIRED | Yes | *string* | The id of the current user.<br />For this hook, the user is expected to be of type [Practitioner](https://www.hl7.org/fhir/practitioner.html).<br />For example, `Practitioner/123`
+`patientId` | REQUIRED | Yes | *string* |  The FHIR `Patient.id` of the current patient in context
+`encounterId` | OPTIONAL | Yes | *string* |  The FHIR `Encounter.id` of the current encounter in context
+`allergyIntolerances` | REQUIRED | No | *object* | DSTU2 - FHIR Bundle of AllergyIntolerance<br/> STU3 - FHIR Bundle of AllergyIntolerance
+
```
Comment (brynrhodes) [inline docs/hooks/allergyintolerance-create.md]: Add `R4 - FHIR Bundle of AllergyIntolerance`

File: docs/hooks/allergyintolerance-create.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+### Examples
+
+### Example (STU3)
```
Comment (brynrhodes) [inline docs/hooks/allergyintolerance-create.md]: Add R4 example

File: docs/hooks/allergyintolerance-create.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+This hook fires during the act of finalizing the entry of a new allergy, such that the decision support returned from the CDS Service can guide the clinician to cancel the addition of the allergy. The context of the hook include the AllergyIntolerance resource that's about to be added to the patient's list of allergies. 
+
+## Context
+
+Field | Optionality | Prefetch Token | Type | Description
+----- | -------- | ---- | ---- | ----
+`userId` | REQUIRED | Yes | *string* | The id of the current user.<br />For this hook, the user is expected to be of type [Practitioner](https://www.hl7.org/fhir/practitioner.html).<br />For example, `Practitioner/123`
+`patientId` | REQUIRED | Yes | *string* |  The FHIR `Patient.id` of the current patient in context
+`encounterId` | OPTIONAL | Yes | *string* |  The FHIR `Encounter.id` of the current encounter in context
+`allergyIntolerance` | REQUIRED | No | *object* | R4/STU3/DSTU2 - FHIR AllergyIntolerance isntance
```
Comment (vadi2) [inline docs/hooks/allergyintolerance-create.md:21]: Small typo:

```suggestion
`allergyIntolerance` | REQUIRED | No | *object* | R4/STU3/DSTU2 - FHIR AllergyIntolerance instance
```

This makes me realise we should do an R5 pass on the spec - something for the todo list.

**Comment by brynrhodes** [comment]
CDS WG voted to approve this proposal: https://confluence.hl7.org/display/CDS/2024-01-10+-+Meeting+Agenda

**Comment by isaacvetter** [comment]
> CDS WG voted to approve this proposal: https://confluence.hl7.org/display/CDS/2024-01-10+-+Meeting+Agenda

@brynrhodes , your request to add an R4 example has been done, but you're marked as requesting changes, which is blocking a normal Merge. Mind approving?

**Review by brynrhodes — APPROVED** [review]

**Pull request merged**